pom.xml

```diff
@@ -243,10 +248,37 @@
             <portName>test.harvester.port</portName>
             <portName>test.db.port</portName>
             <portName>test.s3.port</portName>
+            <portName>test.provider.port</portName>
           </portNames>
         </configuration>
       </plugin>
 
+      <!-- Points the test OAI-PMH records at the thumbnails S3 bucket -->
+      <plugin>
+        <groupId>com.google.code.maven-replacer-plugin</groupId>
+        <artifactId>replacer</artifactId>
+        <version>${replacer.plugin.version}</version>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>replace</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <includes>
+            <include>${project.basedir}/target/test-classes/provider/**/*.xml</include>
```

Comment:
Was there a reason to do this on the output file rather than use the regular Maven resources plugin's filtering to replace a token in the file?